android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -590,13 +591,17 @@ private void hidePositionChooser()
 
   private void initMap(boolean isLaunchByDeepLink)
   {
-    mMapFragment = (MapFragment) getSupportFragmentManager().findFragmentByTag(MapFragment.class.getName());
+    final FragmentManager manager = getSupportFragmentManager();
+    mMapFragment = (MapFragment) manager.findFragmentByTag(MapFragment.class.getName());
     if (mMapFragment == null)
     {
       Bundle args = new Bundle();
       args.putBoolean(MapFragment.ARG_LAUNCH_BY_DEEP_LINK, isLaunchByDeepLink);
-      mMapFragment = (MapFragment) MapFragment.instantiate(this, MapFragment.class.getName(), args);
-      getSupportFragmentManager()
+      final FragmentFactory factory = manager.getFragmentFactory();
+      mMapFragment = (MapFragment)
+          factory.instantiate(getClassLoader(), MapFragment.class.getName());
```

Comment:
```suggestion
      mMapFragment = (MapFragment) factory.instantiate(getClassLoader(), MapFragment.class.getName());
```
+1 к Вите. Не надо лишние строки создавать там, где не нужно.

Reply:
Надо format code сделать на все файлы разово и дальше включить эту проверку. После чего проблема уйдет. Изменений там будет немного.

Reply:
Сейчас пришлось выключить форматтирование при сохранении и каждый раз править руками, потому что иногда в diff попадают formatting изменения, чего происходить не должно.